xcode/Subconscious/Shared/Components/Feed.swift

```diff
@@ -288,9 +305,48 @@ struct FeedModel: ModelProtocol {
                 state: state,
                 environment: environment
             )
+        case let .succeedUpdateAudience(receipt):
+            return detailStackNotification(
+                state: state,
+                environment: environment,
+                action: .succeedUpdateAudience(receipt)
+            )
+        case let .succeedMoveEntry(from, to):
+            return detailStackNotification(
+                state: state,
+                environment: environment,
+                action: .succeedMoveEntry(from: from, to: to)
+            )
+        case let .succeedMergeEntry(parent, child):
+            return detailStackNotification(
+                state: state,
+                environment: environment,
+                action: .succeedMergeEntry(parent: parent, child: child)
+            )
+        case let .succeedSaveEntry(address, modified):
+            return detailStackNotification(
+                state: state,
+                environment: environment,
+                action: .succeedSaveEntry(address: address, modified: modified)
+            )
         }
     }
-
+    
+    static func detailStackNotification(
```

Comment:
This seems somewhat surprisingly named to me. AFAICT, it's not sending up a notification, but forwarding down an intercepted action?

It's also somewhat unusual to have an update fn that takes an action from another context.

I think it might be more straightforward to have these update arms inlined, or given separate static methods. I think that might make them easier to test, or to handle behavior differently in future.

Reply:
I've inlined the `update` calls, but I'm not sure that this makes sense to me:

> It's also somewhat unusual to have an update fn that takes an action from another context.

It's just extracting common functionality in a way that's almost identical to a cursor's update function, perhaps if it were named `forwardWithRefresh` it wouldn't seem so odd? Given the footgun opportunity to omit `.detailStack()` around the forwarded actions I thought _one_ mapping site was safer & more testable (I made this mistake during the refactor).

I suppose I don't see a reason to repeatedly write the same code 4 times upfront, you can always branch one out if/when it's called for.

Reply:
Fair point. If you like, we can leave as-was, but I would prefer a different name, since notification is used elsewhere to mean "sending a message up into another parent context we don't know anything about"